### ISSUE TITLE
Akai MPKmini2 support

### ISSUE THREAD

**Issue by dmedri** [soyersoyer/sysex-controls#6]
Akai MPKmini2 is an other very common small keyboard to play music.
Let's me know if I can help you to support it.

**Comment by soyersoyer** [comment]
I've added the MPKmini2 to the main branch (based on https://github.com/PiOverFour/MPK-M2-editor/blob/master/mpk-m2-editor). But it's completely untested. Would you like to test it?

**Comment by Lesprilibre** [comment]
Yes, i would like to ! I've got a Mk2 somewhere and can give a help !
